Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN apk update && apk upgrade && \
     ${BUILD_DEPENDENCIES} \
     ${APP_DEPENDENCIES} && \
     gem install bundler --no-ri --no-rdoc && \
-    cd ${DCAF_DIR} ; bundle install --without development test && \
+    cd ${DCAF_DIR} ; bundle install --without development && \
```

Comment:
I would go ahead and remove the --without entirely so it's just `bundle install && \`. I've done that in #930 , I'll go ahead and close that issue if you do it here.